src/subscribers/subscribers.jl

```diff
@@ -0,0 +1,16 @@
+module Subscribers
```

Comment:
Ditto previous statement on publisher submodule. Let's get rid of it if we're not using it.